bcda/auth/middleware_test.go

```diff
@@ -47,17 +48,15 @@ func (s *MiddlewareTestSuite) CreateRouter() http.Handler {
 
 func (s *MiddlewareTestSuite) SetupTest() {
 	s.SetupAuthBackend()
-	validClaims := jwt.MapClaims{
-		"sub": "82503A18-BF3B-436D-BA7B-BAE09B7FFD2F",
-		"aco": "DBBD1CE1-AE24-435C-807D-ED45953077D3",
-		"id":  "d63205a8-d923-456b-a01b-0992fcb40968",
-		"iat": time.Now().Unix(),
-		"exp": time.Now().Add(time.Duration(999999999)).Unix(),
+	userID := "82503A18-BF3B-436D-BA7B-BAE09B7FFD2F"
+	acoID := "DBBD1CE1-AE24-435C-807D-ED45953077D3"
```

Comment:
I see that BCDA-1040 will remove the userID from the alpha tokens.  Not here, but should we also align alpha and Okta as to the identifier being returned for the ACO?

Reply:
If you mean for constructing the token, I think the answer there will be that the plugins need to have a testToken() method (not exported) that will generate test tokens. That's one of the reasons I focused so much on getting a standard pattern going for creating them. If we do that, the same tests will work for either plugin.